README.rst

```diff
@@ -122,6 +122,16 @@ Then run:
 RESTful API
 ===========
 
+API Authentication
+##################
+
+The API authentication is based on session based authentication via Django REST framework.
+This authentication scheme uses Django's default session backend for authentication.
+
+.. code-block:: text
+
+    <api-url> -H 'Cookie: csrftoken=<csrftoken>; sessionid=<session-id>'
```

Comment:
no man, this is not good. the API must work without the CSRF mechanism, this is not a webpage.

Ensure the API works without the CSRF token